index.html

```diff
@@ -15,40 +15,42 @@
       rel="stylesheet"
     />
     <link rel="stylesheet" href="style.css" />
-<!-- Global site tag (gtag.js) - Google Analytics -->
-<script async src="https://www.googletagmanager.com/gtag/js?id=G-B5S8VEZHQY"></script>
-<script>
-  window.dataLayer = window.dataLayer || [];
-  function gtag(){dataLayer.push(arguments);}
-  gtag('js', new Date());
+    
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-B5S8VEZHQY"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
 
-  gtag('config', 'G-B5S8VEZHQY');
-</script>
+      gtag('config', 'G-B5S8VEZHQY');
+    </script>
   </head>
-  <body>
+  <body id="body">
     <header class="header">
       <a href="#" class="logo">My Logo</a>
 
       <nav>
-        <i class="fas fa-bars mobile-menu"></i>
+        <i class="fas fa-bars mobile-menu" id="mobile-menu"></i>
         <ul class="desktop-menu">
-            <li>
-                <a href="#portfolio">Portfolio</a>
-            </li>
-            <li>
-                <a href="#about">About</a>
-            </li>
-            <li>
-                <a href="#contact">Contact</a>
-            </li>
+          <li>
+            <a href="#portfolio">Portfolio</a>
+          </li>
+          <li>
+            <a href="#about">About</a>
+          </li>
+          <li>
+            <a href="#contact">About</a>
+          </li>
```

Comment:
The text on the `Contact` link should be `Contact` and not `About` as seen in line 43.
Kindly fix this.